### PR TITLE
Prepare release 0.3.0

### PR DESCRIPTION
**Pull request by dkhalanskyjb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 0.3.0
+
+#### Features
+
+- Added `iosSimulatorArm64`, `watchosSimulatorArm64`, `tvosSimulatorArm64`, `macosArm64` target support ([PR#142](https://github.com/Kotlin/kotlinx-datetime/pull/142)).
```
Comment (ilya-g) [inline CHANGELOG.md:7]: Better to reference the corresponding issue, if there is one.

File: README.md [hunk 2]
```diff
@@ -36,13 +36,14 @@ The library provides the basic set of types for working with date and time:
...
 - `LocalDate` to represent the components of date only;
-- `TimeZone` and `ZoneOffset` provide time zone information to convert between `Instant` and `LocalDateTime`;
+- `TimeZone` and `FixedOffsetTimeZone` provide time zone information to convert between `Instant` and `LocalDateTime`;
 - `Month` and `DayOfWeek` enums;
 - `DateTimePeriod` to represent a difference between two instants decomposed into date and time units;
 - `DatePeriod` is a subclass of `DateTimePeriod` with zero time components,
 it represents a difference between two LocalDate values decomposed into date units.
 - `DateTimeUnit` provides a set of predefined date and time units to use in arithmetic operations on `Instant` and `LocalDate`. 
-  
+- `UtcOffset` represents the UTC offset of an unspecified time zone at an unspecified moment.
```
Comment (ilya-g) [inline README.md:45]: That "unspecified" is rather confusing.

File: CHANGELOG.md (hunk 1 above)
Comment (dkhalanskyjb) [reply CHANGELOG.md]: Fixed.

File: README.md (hunk 2 above)
Comment (dkhalanskyjb) [reply README.md]: Changed. Is the new version better?

**Review by ilya-g — APPROVED** [review]

**Pull request merged**